exchanges/refocus/src/refocusExchange.ts

```diff
@@ -6,14 +6,19 @@ export const refocusExchange = (): Exchange => {
     const watchedOperations = new Map<number, Operation>();
     const observedOperations = new Map<number, number>();
 
-    window.addEventListener('focus', () => {
-      watchedOperations.forEach(op => {
-        client.reexecuteOperation(
-          client.createRequestOperation('query', op, {
-            requestPolicy: 'cache-and-network',
-          })
-        );
-      });
+    window.addEventListener('visibilitychange', () => {
+      if (
+        typeof document !== 'object' ||
```

Comment:
At the moment this will only execute on the server with this check

```suggestion
        typeof window !== 'undefined' ||
```

Reply:
No the condition is an or 😅 so it says "document isn't an object (server) OR visible"